components/logins/src/sync/engine.rs

```diff
@@ -30,26 +29,15 @@ pub struct LoginsSyncEngine {
     pub store: Arc<LoginStore>,
     pub scope: SqlInterruptScope,
     pub staged: RefCell<Vec<IncomingBso>>,
-    // It's unfortunate this is an Option<>, but tricky to change because sometimes we construct
-    // an engine for, say, a `reset()` where this isn't needed or known.
-    encdec: Option<EncryptorDecryptor>,
```

Comment:
Very happy to lose this  :clinking_glasses:

Reply:
\o/